--- origsrc/e2fsprogs-1.44.5/lib/uuid/uuidd.h	2018-12-15 23:39:33.000000000 -0600
+++ src/e2fsprogs-1.44.5/lib/uuid/uuidd.h	2018-12-31 16:58:24.713953100 -0600
@@ -35,7 +35,7 @@
 #ifndef _UUID_UUIDD_H
 #define _UUID_UUIDD_H
 
-#define UUIDD_DIR		"/var/lib/libuuid"
+#define UUIDD_DIR		"/var/run/uuidd"
 #define UUIDD_SOCKET_PATH	UUIDD_DIR "/request"
 #define UUIDD_PIDFILE_PATH	UUIDD_DIR "/uuidd.pid"
 #define UUIDD_PATH		"/usr/sbin/uuidd"
@@ -48,6 +48,8 @@
 #define UUIDD_OP_BULK_RANDOM_UUID	5
 #define UUIDD_MAX_OP			UUIDD_OP_BULK_RANDOM_UUID
 
+#define uuid__generate_time __uuid_generate_time
+#define uuid__generate_random __uuid_generate_random
 extern void uuid__generate_time(uuid_t out, int *num);
 extern void uuid__generate_random(uuid_t out, int *num);
 
--- origsrc/e2fsprogs-1.44.5/misc/uuidd.8.in	2018-12-15 23:39:33.000000000 -0600
+++ src/e2fsprogs-1.44.5/misc/uuidd.8.in	2019-01-01 00:24:54.560632800 -0600
@@ -64,11 +64,11 @@ UUID's.
 .TP
 .BI \-p  " pidfile"
 Specify the pathname where the pid file should be written.  By default,
-the pid file is written to /var/lib/libuuid/uuidd.pid.
+the pid file is written to /var/run/uuidd/uuidd.pid.
 .TP
 .BI \-s " socketpath"
 Specify the pathname used for the unix-domain socket used by uuidd.  By
-default, the pathname used is /var/lib/libuuid/request.  This is primarily
+default, the pathname used is /var/run/uuidd/request.  This is primarily
 for debugging purposes, since the pathname is hard-coded in the libuuid
 library.
 .TP
